assets/training/model_evaluation/environment/context/conda.yaml

```diff
@@ -25,4 +25,8 @@ dependencies:
   - statsmodels==0.11.0
   - lightgbm==3.2.1
   - msrest==0.6.21
-  - mltable=={{latest-pypi-version}}
\ No newline at end of file
+  - mltable=={{latest-pypi-version}}
+  - cryptography==41.0.2
```

Comment:
why are we adding this explicitly? this was never a dependency in ME components. @skasturi

Reply:
Done. Reverted model evaluation.